### PR TITLE
Integration tests

### PR DESCRIPTION
**Pull request by vogdb**
Depends on #53

### CODE REVIEW

**Comment by vogdb** [comment]
@zazabe please review. A commit-by-commit approach will be easier.

**Comment by vogdb** [comment]
@zazabe I hope that you are reviewing this :)

**Comment by zazabe** [comment]
got this error with `vagrant up` / `vagrant provision`:
```
$ vagrant provision
==> default: Installing Puppet modules in "puppet" with Librarian-Puppet...
==> default: Running provisioner: puppet...
==> default: Running Puppet with environment development...
==> default: stdin: is not a tty
==> default: Warning: Config file /etc/puppet/hiera.yaml not found, using Hiera defaults
==> default: Error: ::facts is not a hash or array when accessing it with lsbdistcodename at /tmp/vagrant-puppet/modules-f857ccb0d9754d1d6058c59b66e616a4/apt/manifests/init.pp:17 on node janus-gateway-js.dev.cargomedia.ch
==> default: Error: ::facts is not a hash or array when accessing it with lsbdistcodename at /tmp/vagrant-puppet/modules-f857ccb0d9754d1d6058c59b66e616a4/apt/manifests/init.pp:17 on node janus-gateway-js.dev.cargomedia.ch
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
```

**Comment by vogdb** [comment]
hm, @ppp0 ?

**Comment by ppp0** [comment]
update your vagrant box!

**Comment by zazabe** [comment]
yes sir !

File: test/integration/rtpbroadcast.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        assert.equal(list[0]['id'], mountpointId);
+        return rtpbroadcastPlugin.destroy(mountpointId);
+      })
+      .then(function(response) {
+        assert.equal(response['plugindata']['data']['destroyed'], mountpointId);
+        done();
+      });
+  });
+
+  it('creates, watches, pauses, starts, stops and destroys', function(done) {
```
Comment (zazabe) [inline test/integration/rtpbroadcast.js]: maybe some assertions could be added for this test (`addstream` triggered, `PeerConnection` state,...)

File: README.md [hunk 2]
```diff
@@ -509,3 +509,6 @@ The library is available for Node and Browser environment. In Browser it is decl
 
     Creates a new instance of ConnectionError. Extends JanusError.
     * `janusMessage` {Object} message that caused the error.
+
+## Tests
+There are unit and integration tests. Unit tests are automated and included into CI. Integration tests are not. To check integration tests you need to manually load `test/integration/index.html` into the latest version of Chrome and verify it manually. To run unit tests just use `npm test`.
```
Comment (zazabe) [inline README.md]: Specify that integration tests have to be run with `http://localhost` to bypass the `insecure origins` warning on Chrome.

**Comment by zazabe** [comment]
Nice! it works and looks very good!
Please, just check my 2 small comments above: https://github.com/cargomedia/janus-gateway-js/pull/55#discussion_r69439259, https://github.com/cargomedia/janus-gateway-js/pull/55#discussion_r69439992

**Comment by vogdb** [comment]
@zazabe please review

**Comment by zazabe** [comment]
lgtm

**Pull request merged**